src/qml/components/PeersIndicator.qml

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+// The PeersIndicator component.
+// See reference design: https://bitcoindesign.github.io/Bitcoin-Core-App/assets/images/block-clock-connection-states-big.png
```

Comment:
Perhaps every component with a one to one link to a design file should have a link for easy reference to the design spec?